### DISCUSSION TITLE
Update Delay - 2/4/2023

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#235] [Announcements]
Hello everyone,

As I mentioned before, this month is going to be exam month for most of the University student. I am no exception. So I will probably going to take a short break from Eden. This break will be around 2 weeks. Support will still continue. 

Don't worry! I have no plan on give up on developing Eden.

**Comment by HappyAreaBean** [comment]
good luck bro

**Comment by Stainded** [comment]
Good luck man.. =)
